src/js/components/SearchForm.js

```diff
@@ -225,8 +225,12 @@ export default class SearchForm extends React.PureComponent {
       }))
     );
 
-  handleSearchChange = ({ target: { value } }) => {
-    this.setQueryAs(value)();
+  handleSearchChange = ({ target }) => {
+    var cursorStart = target.selectionStart,
```

Comment:
Could you please use `const` and have separate statements for variable declarations?

Reply:
Fixed.